android/uhabits-core/src/test/java/org/isoron/uhabits/core/io/HabitsCSVExporterAndImporterTest.java

```diff
@@ -77,6 +90,39 @@ public void testExportCSV() throws IOException
         assertPathExists("Scores.csv");
     }
 
+    @Test
+    public void testImportCSV() throws IOException {
+        List<Habit> selected = new LinkedList<>();
+        for (Habit h : habitList) selected.add(h);
+
+        HabitsCSVExporter exporter =
```

Comment:
Ah. Now I see why you named it HabitsCSVImporter. I still think LoopCSVImporter is the better name though.